gtest/src/manager.rs

```diff
@@ -128,7 +130,7 @@ pub(crate) struct ExtManager {
     // State
     pub(crate) actors: BTreeMap<ProgramId, (Actor, Balance)>,
     pub(crate) dispatch_queue: VecDeque<Dispatch>,
-    pub(crate) mailbox: BTreeMap<ProgramId, Vec<Message>>,
+    pub(crate) actor_to_mailbox: HashMap<ProgramId, Mailbox>,
```

Comment:
Please, revert it. Seems bad

Reply:
User-API shouldn't change inner data-storing system. Providing separated struct `Mailbox` seems nice for me, but it should be built lazy, on-query of function